program/src/processor.rs

```diff
@@ -463,17 +565,13 @@ pub fn process_withdraw_inactive_stake(
     let mut observed_total = Lamports(0);
     let mut excess_removed = Lamports(0);
     let mut stake_accounts = accounts.stake_accounts.iter();
-    let begin = validator.entry.stake_accounts_seed_begin;
-    let end = validator.entry.stake_accounts_seed_end;
+    let begin = validator.entry.stake_seeds.begin;
+    let end = validator.entry.stake_seeds.end;
 
     // Visit the stake accounts one by one, and check how much SOL is in there.
     for seed in begin..end {
```

Comment:
We now need to walk over all unstake accounts as well here. Maybe we can extract the loop body into a function so we can reuse it for stake accounts and unstake accounts. One thing that we need to change is that for unstake accounts, it can happen that they are 100% inactive, and if we withdraw 100%, then we need to bump the seed for that account. (It doesn’t need to be a special case for unstake, it will just not happen for regular stake accounts).